frontend-react/package.json

```diff
@@ -152,6 +152,7 @@
     },
     "resolutions": {
         "react-scripts/**/node-forge": "^1.0.0",
+        "react-scripts/**/url-parse": "^1.5.6",
```

Comment:
Was there a build error before adding the resolution?

Reply:
I don't believe there was a build error. This is to resolve a Dependabot warning.

Reply:
Ah got it.